css/style.css

```diff
@@ -0,0 +1,2932 @@
+  line-height: 16px;
+  margin-right: 0;
+  padding-bottom: 3px;
+}
+
+.button {
+  display: flex;
+  align-items: center;
+  padding: 10px 28px;
+  border: 2px solid #fff;
+  font-size: 13px;
+  text-transform: uppercase;
+  background-color: #fff;
+}
+
+.button__red {
+  border: 2px solid #e51937;
+  color: #e51937;
+  padding-top: 9px;
+  padding-bottom: 8px;
+  padding-left: 40px;
+  padding-right: 42px;
+  margin-bottom: 17px;
+  margin-right: 23px;
+}
+
+.button img {
+  margin-right: 8px;
+}
+
+.middle {
+  margin-top: 31px;
+}
+
+.middle__inner {
+  background: url(../img/svg/tesla-model-s-2.png) no-repeat;
+  background-position: 0 60px;
+  background-size: auto;
+}
+
+.middle__fon {
+  display: flex;
+  justify-content: space-between;
+}
+
+.middle__title {
+  min-width: 300px;
+  font-weight: 600;
+  font-size: 70px;
+  line-height: 85px;
+  color: #E51937;
+  margin-left: 80px;
+  margin-top: 65px;
+}
+
+.middle__info {
+  display: flex;
+  flex-direction: column;
+  color: #fff;
+  background-color: #E51937;
+  max-width: 440px;
+  height: 900px;
+}
+
+.middle__content {
+  margin-left: 52px;
+}
+
+.middle__info-title {
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 37px;
+  margin-top: 51px;
+  margin-bottom: 12px;
+}
+
+.middle__info-text {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  margin-top: 11px;
+  margin-right: 70px;
+  margin-bottom: 50px;
+}
+
+.middle__row {
+  display: flex;
+  flex-direction: column;
+  padding-top: 5px;
+  padding-left: 45px;
+  padding-right: 150px;
+}
+
+.middle__btn {
+  margin-top: 90px;
+  margin-left: 8px;
+}
+
+.middle__column {
+  display: flex;
+  align-items: center;
+  margin-bottom: 6px;
+}
+
+.middle__column:nth-child(3) {
+  margin-top: 5px;
+}
+
+.middle__screen {
+  padding-top: 18px;
+  padding-left: 7px;
+  padding-bottom: 3px;
+}
+
+.middle__wifi {
+  padding-top: 0;
+  padding-bottom: 0;
+  margin-left: 3px;
+  margin-top: 1px;
+}
+
+.middle__bag {
+  padding-right: 4px;
+  padding-top: 18px;
+  padding-left: 7px;
+  padding-bottom: 0;
+}
+
+.middle__item {
+  height: 90px;
+}
+
+.middle__first {
+  padding-top: 5px;
+  padding-left: 0px;
+}
+
+.middle__first-big {
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+  padding-left: 20px;
+  padding-bottom: 0;
+  padding-right: 7px;
+}
+
+.middle__first-small {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 24px;
+  padding-left: 0;
+}
+
+.middle__first-title{
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 18px;
+  padding-left: 20px;
+  padding-bottom: 0;
+  padding-top: 8px;
+}
+
+.middle__first-text {
+  font-size: 12px;
+  line-height: 15px;
+  padding-left: 20px;
+  padding-bottom: 10px;
+}
+
+.middle__text {
+  font-size: 12px;
+  line-height: 15px;
+  padding-left: 20px;
+}
+
+.middle__btn {
+  margin-top: 95px;
+  margin-left: 12px;
+}
+
+.button__btn {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  border: 2px solid #E51937;;
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)